pyiron/lammps/control.py

```diff
@@ -250,8 +250,10 @@ def pressure_to_lammps(self, pressure, rotation_matrix):
 
     def calc_minimize(
```

Comment:
This function is never called from the user directly, so we do not need `e_tol` and `f_tol` here.

Reply:
Shouldn’t it be a private function then? In general I was not really sure which functions can be used by the user and which ones cannot be.

Reply:
The issue is that pep8 defines private functions as those which should only be used within the same class and derived. But in pyiron we have three levels: 

- user level - what is used in the notebook
- developer level - what is used in derived classes but should never be called from the user directly
- class level - what is only used within one class but not in other classes - _private functions_